engine/table/src/main/java/io/deephaven/engine/table/impl/hierarchical/TreeNodeOperationsRecorder.java

```diff
@@ -29,16 +29,17 @@ class TreeNodeOperationsRecorder extends BaseNodeOperationsRecorder<TreeTable.No
     private final Collection<? extends WhereFilter> recordedFilters;
 
     TreeNodeOperationsRecorder(@NotNull final TableDefinition definition) {
-        this(definition, List.of(), List.of(), List.of(), List.of());
+        this(definition, List.of(), List.of(), List.of(), List.of(), List.of());
     }
 
     private TreeNodeOperationsRecorder(
             @NotNull final TableDefinition definition,
             @NotNull final Collection<? extends SelectColumn> recordedFormats,
             @NotNull final Collection<SortColumn> recordedSorts,
             @NotNull final Collection<? extends SelectColumn> recordedAbsoluteViews,
+            @NotNull final Collection<? extends SelectColumn> recordedViewColumns,
```

Comment:
The tree table uses an interface without the UpdateViewRecorder, so it doesn't seem appropriate to include it here; which is beyond the scope of the original ticket.